Assets/Scripts/UI/UIScrollers/HorizontalScroller.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using DN.SceneManagement.Data;
+
+namespace DN.UI
+{
+    public class HorizontalScroller : MonoBehaviour
+    {
+        [SerializeField] private LevelsData levelsData;
+
+        public List<Button> btn = new List<Button>();
```

Comment:
Een `List` mag niet public, dit zorgt er voor dat andere classes deze lijst kunnen modifyen of accessen.